src/librustdoc/html/static/js/main.js

```diff
@@ -1240,8 +1263,10 @@ function preLoadCss(cssUrl) {
         }
     }
 
-    onEachLazy(document.getElementsByClassName("tooltip"), e => {
-        e.onclick = () => {
+    window.rustdocConfigureTooltip = e => {
+        e.addEventListener("click", ev => {
```

Comment:
We originally made it `onclick` to prevent to add more `click` events iirc. But beyond that, is this change necessary? You still get the event as first argument even with `.onclick = ev => {`.